tests/common.cpp

```diff
@@ -318,6 +318,25 @@ result_t validateFloatEpsilon(__m128 a,
     return TEST_SUCCESS;
 }
 
+result_t validateFloatError(__m128 a,
+                            float f0,
+                            float f1,
+                            float f2,
+                            float f3,
+                            float err)
+{
+    const float *t = (const float *) &a;
+    float df0 = fabsf((t[0] - f0) / f0);
+    float df1 = fabsf((t[1] - f1) / f1);
+    float df2 = fabsf((t[2] - f2) / f2);
+    float df3 = fabsf((t[3] - f3) / f3);
```

Comment:
I wonder the purpose of the division.

Reply:
I mean, I am not convinced by the description `I think an error is more straightforward`.
Could you give more detail or your thought of the validation?

Reply:
I think the error of the two input is more representative than the difference of the two input, since the two input may be a relatively small number, in which case, a small difference would become relatively big, and the test can't pass in this situation.

Reply:
In contrast, how about the situation when the two inputs are large number (float maximum: 3.4×10^38)?

Reply:
I think we should use a proportional value which varies against the inputs numbers to validate the test result. Therefore, under the situation that the 2 inputs are large numbers, we still guarantee the accuracy of the result.
Instead, if using an absolute difference of validate the test result, than the accuracy would vary against the values of the input numbers which is inappropriate and make the result unpredictable.